src/els_code_action_provider.erl

```diff
@@ -0,0 +1,183 @@
+-module(els_code_action_provider).
+
+-behaviour(els_provider).
+
+-export([ handle_request/2
+        , is_enabled/0
+        , options/0
+        ]).
+
+-include("erlang_ls.hrl").
+
+%%==============================================================================
+%% els_provider functions
+%%==============================================================================
+
+-spec is_enabled() -> boolean().
+is_enabled() -> true.
+
+%% /**
```

Comment:
Please use `@edoc` syntax for docs.

Reply:
Note: the tag is `@doc`, not `@edoc`.  I am correcting this in the rest of  the app too.